### PR TITLE
Add annotations to a record

### PR DESCRIPTION
**Pull request by galanoff**
It is possible to define 10 annotation regexps (annotation_match1 to
annotation_match1 10) to match kube's annotation keys. Mathed entries
will be added to a record.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ This must used named capture groups for `container_name`, `pod_name` & `namespac
 * `de_dot_separator` - separator to use if `de_dot` is enabled (default: `_`)
 * `use_journal` - If false (default), messages are expected to be formatted and tagged as if read by the fluentd in\_tail plugin with wildcard filename.  If true, messages are expected to be formatted as if read from the systemd journal.  The `MESSAGE` field has the full message.  The `CONTAINER_NAME` field has the encoded k8s metadata (see below).  The `CONTAINER_ID_FULL` field has the full container uuid.  This requires docker to use the `--log-driver=journald` log driver.
 * `container_name_to_kubernetes_regexp` - The regular expression used to extract the k8s metadata encoded in the journal `CONTAINER_NAME` field (default: `'^k8s_(?<container_name>[^\.]+)\.(?<container_hash>[a-z0-9]{8})_(?<pod_name>[^_]+)_(?<namespace>[^_]+)_(?<pod_id>[^_]+)_(?<pod_randhex>[a-z0-9]{8})$'`)
+* `annotation_match<N>` - The regular expressions used to match annotation field names. Where N = 1..10.  Matched annotations are added to a log record.
```
Comment (jimmidyson) [inline README.md]: Can this be an array param instead So would be used like:

```
 annotation_match [
  "^custom.+",
  "^next.+"
]
```

**Comment by galanoff** [comment]
Hi Jimmy,

Could you please take a look at the latest commit? It includes the change you've mentioned.

**Comment by jimmidyson** [comment]
LGTM - thanks! Sorry for the slow response.

**Comment by jimmidyson** [comment]
FYI just releases 0.24.0 with this change.

**Pull request merged**